website/docs/reference/warehouse-setups/vertica-setup.md

```diff
@@ -39,7 +39,7 @@ If you're interested in contributing, check out the source code for each reposit
 
 <h2> Installing {frontMatter.meta.pypi_package} </h2>
 
-pip is the easiest way to install the adapter: <code>pip install {frontMatter.meta.pypi_package}</code>
+pip is the easiest way to install the adapter: <code>pip install dbt-vertica</code>
```

Comment:
This seems to be rendering fine?
<img width="871" alt="image" src="https://user-images.githubusercontent.com/8158673/212810282-af66f708-232d-4912-8f53-eb95fa2bc406.png">

```suggestion
pip is the easiest way to install the adapter: <code>pip install {frontMatter.meta.pypi_package}</code>
```

Reply:
![Screenshot from 2023-01-18 00-02-07](https://user-images.githubusercontent.com/25935995/212982258-788a5a77-ad0b-4626-84b5-c419c50a64d8.png)
 @dataders  doesnt seems to be on this link : https://docs.getdbt.com/reference/warehouse-setups/vertica-setup